embark/workers/views.py

```diff
@@ -353,7 +357,8 @@ def worker_soft_reset(request, worker_id, configuration_id=None):
     """
     try:
         if not configuration_id and not worker_id:
```

Comment:
Es wäre wahrscheinlich besser ohne einen check für die config id weil die ja eh optional ist

```suggestion
        if not worker_id:
```